src/main/java/com/softserveinc/speakukrainian/pageobjects/components/addCenterVisitor/AddLocationModal.java

```diff
@@ -31,14 +32,14 @@ public class AddLocationModal extends BasePage {
             $$x("#/html/body/div[9]/div/div/div[2]/div[1]/div/div/div");
 
     public static final SelenideElement ADD_BUTTON =
-            $x("button[class='ant-btn css-18v7s9k ant-btn-default flooded-button add-club-content-next']");
```

Comment:
The locator is invalid, does not match the Xpath syntax, and uses a dynamic class for searching